.github/workflows/post.yml

```diff
@@ -133,3 +132,14 @@ jobs:
           docker manifest push quay.io/open-cluster-management/work:latest
           docker manifest push quay.io/open-cluster-management/placement:latest
           docker manifest push quay.io/open-cluster-management/addon-manager:latest
+  trigger-clusteradm-e2e:
+    needs: [ images, image-manifest ]
+    name: trigger clusteradm e2e
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.OCM_BOT_PAT }}
```

Comment:
Hi, @mikeshng does OCM_BOT_PAT has write permission on `Contents`? The permission is needed to send a repository-dispatch event.
![image](https://github.com/open-cluster-management-io/ocm/assets/11444421/6bc00f95-7326-4b86-84ab-e181855e0aa3)